scripts/create.sh

```diff
@@ -22,14 +22,14 @@ set -e
 ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." && pwd )"
 
 # Include the user set variables
-# shellcheck source=properties.env
+# shellcheck disable=SC1091,SC1090
```

Comment:
Why did you add the disables?  Can we use the `source`?

Reply:
Seemed not to work, but I saw it worked on yours so let me take a look again.